install_config/persistent_storage/persistent_storage_iscsi.adoc

```diff
@@ -31,9 +31,8 @@ storage provider.
 [[provisioning-iscsi]]
 
 == Provisioning
-Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in {product-title}. All that is required for iSCSI is iSCSI target
-portal, valid iSCSI IQN, valid LUN number, and filesystem type, and the
+Verify that the storage exists in the underlying infrastructure before mounting it as a volume in {product-title}. All that is required for iSCSI is iSCSI target
+portal, valid iSCSI Qualified Name (IQN), valid LUN number, and filesystem type, and the
```

Comment:
I think there's "the" and "and" things above? Maybe:

"All that is required for iSCSI is the iSCSI target portal, a valid iSCSI Qualified Name (IQN), a valid LUN number, the filesystem type, and the `*PersistentVolume*` API."

?